packages/wallet-sdk/src/core/communicator/KeysPopupCommunicator.ts

```diff
@@ -0,0 +1,67 @@
+import { LIB_VERSION } from '../../version';
+import { ConfigMessage, Message } from '../message';
+import { closePopup, openPopup } from './PopUpCommunicator';
+import { CB_KEYS_URL } from ':core/constants';
+
+export class KeysPopupCommunicator {
```

Comment:
might be useful to check out some individual commits:
- https://github.com/coinbase/coinbase-wallet-sdk/pull/1260/commits/34b710da2a10ff78a01d9aceefe3ca3f05b04544
- https://github.com/coinbase/coinbase-wallet-sdk/pull/1260/commits/a2f8c47f5f166c5c12ab2e9e9b7558535127d44c